src/docs/product/performance/web-vitals.mdx

```diff
@@ -0,0 +1,67 @@
+
+
+| Web Vital                                                 | Availability                  | Failure Threshold |
+| --------------------------------------------------------- | ----------------------------- | ----------------- |
+| [First Paint](#first-paint-fp)                            | Chromium                      | 3 seconds         |
+| [First Contentful Paint](#first-contentful-paint-fcp)     | Chromium                      | 3 seconds         |
+| [Largest Contentful Paint](#largest-contentful-paint-lcp) | Chromium                      | 4 seconds         |
+| [First Input Delay](#first-input-delay-fid)               | Chromium, Firefox, and Safari | 300 milliseconds  |
+| [Cumulative Layout Shift](#cumulative-layout-shift-cls)   | Chromium                      | 0.25              |
+
+
+## Baseline Value
+
+The baseline value for each web vital is the 75th percentile of the observed data points. It is compared against a failure threshold to give a pass or fail for each web vital.
+
+<Note>
+Web Vital terminologies such as "75th percentile" and "failure threshold" apply to the specific Web Vitals. Not to be confused with similar terminology for Performance Metrics.
+</Note>
+
+## Distribution Histogram
+
+A histogram visualization showing the data distribution can help you identify and diagnose frontend performance problems by revealing anomalies.
+
+By default, outliers will be excluded from the histograms to provide a more informative view of these vitals. Outliers are determined using the [upper outer fence](https://www.itl.nist.gov/div898/handbook/prc/section1/prc16.htm) as the upper bound, and any data points above the upper bound are deemed an outlier.
+
+If you notice a region of interest on any of the histograms, click and drag over the area to zoom in for a more detailed view. You may also want to see more information related to the transactions in the histograms. Click "Open in Discover" beneath the web vital of choice to build a custom query for further investigation. For more details, see the full documentation for the Discover [Query Builder](/product/discover-queries/query-builder/).
+
+If you wish to see all of the data available, open the dropdown and click "View All". You will likely see extreme outliers when you click "View All". You can click and drag over an area to zoom in for a more detailed view.
+
+## Vitals
+
+Vitals measure render time, response time, and the sum of shifting rendering processes. Each data point provides insights about the overall performance of your application.
+
+### First Paint (FP)
+
+First Paint (FP) measures the amount of time the first pixel takes to appear in the viewport, rendering any visual change from what was previously displayed. This may be in any form from the document object model (DOM), such as background color, canvas, or image. FP helps developers understand if anything unexpected is happening to render the page.
```

Comment:
```suggestion
First Paint (FP) measures the amount of time the first pixel takes to appear in the viewport, rendering any visual change from what was previously displayed. This may be in any form from the document object model (DOM), such as non-default background, canvas, or image. FP helps developers understand if anything unexpected is happening to render the page.
```

reference: https://w3c.github.io/paint-timing/#first-paint

Reply:
Going to leave this as just

> background color